On windows, the app crashes when I minimize it

### Check for existing issues

- [x] Completed

### Describe the bug / provide steps to reproduce it

If I just switch to another application, then when I return to zed, it continues to function, but if I minimize it and expand it, it stops responding to any actions other than the window control buttons.
If I expand it to full screen (using the window control button at the top of the zed window), it will start functioning again.
If i collapse it again, everything will repeat itself.

### Environment

Windows 11 24H2   
AMD Ryzen 2700 Eight-Core (x64)
16gb ram
I have compiled zed from the source code   
I have not installed any plugins on it

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

<details><summary>Zed.log</summary>
```
[Zed.log](https://github.com/user-attachments/files/18049222/Zed.log)
```
</details>

I have the same issue. Here's my log of opening zed, minimizing and bringing it back to front (frozen at this point), then closing it.
        <details><summary>Zed.log</summary>
```
2024-12-08T13:24:41.1965219+09:00 [INFO] ========== starting zed ==========
2024-12-08T13:24:41.4100335+09:00 [INFO] Use Segoe UI as UI font.
2024-12-08T13:24:41.4115949+09:00 [INFO] Opening main db
2024-12-08T13:24:41.4203735+09:00 [INFO] Opening main db
2024-12-08T13:24:41.4304223+09:00 [INFO] Using git binary path: None
2024-12-08T13:24:41.4315947+09:00 [ERROR] SHELL environment variable is not assigned so we can't source login environment variables

Caused by:
    environment variable not found
2024-12-08T13:24:41.4843874+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.4872606+09:00 [INFO] No prompt template overrides directory found at C:\Users\Kille\AppData\Local\Zed\prompt_overrides. Using built-in prompts.
2024-12-08T13:24:41.4879256+09:00 [INFO] extensions updated. loading 5, reloading 0, unloading 0
2024-12-08T13:24:41.4998366+09:00 [INFO] set status on client 0: Authenticating
2024-12-08T13:24:41.5002471+09:00 [INFO] set status on client 260961: Connecting
2024-12-08T13:24:41.5008604+09:00 [ERROR] Error { kind: Generic("Input watch path is neither a file nor a directory."), paths: [] }
2024-12-08T13:24:41.5019159+09:00 [ERROR] Error { kind: Generic("Input watch path is neither a file nor a directory."), paths: [] }
2024-12-08T13:24:41.5022281+09:00 [INFO] Opening main db
2024-12-08T13:24:41.5163412+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.5197882+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.5223621+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.5247584+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.5286404+09:00 [ERROR] Access is denied. (os error 5)
2024-12-08T13:24:41.6045264+09:00 [INFO] Enabling Vulkan Portability
2024-12-08T13:24:41.6046146+09:00 [INFO] Enabling color space support
2024-12-08T13:24:41.7085988+09:00 [INFO] Adapter: "NVIDIA GeForce GTX 1650 Ti"
2024-12-08T13:24:41.7087446+09:00 [INFO] No ray tracing extensions are supported
2024-12-08T13:24:41.7386884+09:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-12-08T13:24:41.7890135+09:00 [WARN] Requested size 1x1 is outside of surface capabilities
2024-12-08T13:24:41.7891078+09:00 [ERROR] No composite alpha flag for transparency: OPAQUE
2024-12-08T13:24:41.7902281+09:00 [INFO] Using surface present mode MAILBOX
2024-12-08T13:24:41.7953429+09:00 [WARN] Unable to forbid exclusive full screen
2024-12-08T13:24:41.8110251+09:00 [INFO] Creating a descriptor pool for at most 16 sets
2024-12-08T13:24:41.8111251+09:00 [INFO] Creating a descriptor pool for at most 16 sets
2024-12-08T13:24:41.8112072+09:00 [INFO] Initializing Blade pipelines for surface SurfaceInfo { format: Bgra8UnormSrgb, alpha: Ignored }
2024-12-08T13:24:41.813871+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.814283+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8160233+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8164759+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8172176+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8175836+09:00 [INFO] Skip function Some("to_device_position_impl")
2024-12-08T13:24:41.8176667+09:00 [INFO] Skip function Some("to_device_position")
2024-12-08T13:24:41.8177437+09:00 [INFO] Skip function Some("to_device_position_transformed")
2024-12-08T13:24:41.8178206+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8179885+09:00 [INFO] Skip function Some("blend_color")
2024-12-08T13:24:41.8191999+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8198843+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8202354+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8210773+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8219437+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8225189+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8228425+09:00 [INFO] Skip function Some("to_device_position_impl")
2024-12-08T13:24:41.8229871+09:00 [INFO] Skip function Some("to_device_position")
2024-12-08T13:24:41.8230811+09:00 [INFO] Skip function Some("to_device_position_transformed")
2024-12-08T13:24:41.8231613+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8233474+09:00 [INFO] Skip function Some("blend_color")
2024-12-08T13:24:41.8275018+09:00 [ERROR] No composite alpha flag for transparency: OPAQUE
2024-12-08T13:24:41.8275968+09:00 [INFO] Using surface present mode MAILBOX
2024-12-08T13:24:41.830058+09:00 [WARN] Unable to forbid exclusive full screen
2024-12-08T13:24:41.8359167+09:00 [INFO] Using surface present mode MAILBOX
2024-12-08T13:24:41.8390576+09:00 [WARN] Unable to forbid exclusive full screen
2024-12-08T13:24:41.8408638+09:00 [INFO] Initializing Blade pipelines for surface SurfaceInfo { format: Bgra8UnormSrgb, alpha: Ignored }
2024-12-08T13:24:41.8432374+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8436056+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8442456+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8445842+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8451343+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8454468+09:00 [INFO] Skip function Some("to_device_position_impl")
2024-12-08T13:24:41.8455267+09:00 [INFO] Skip function Some("to_device_position")
2024-12-08T13:24:41.8456032+09:00 [INFO] Skip function Some("to_device_position_transformed")
2024-12-08T13:24:41.8456797+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8458306+09:00 [INFO] Skip function Some("blend_color")
2024-12-08T13:24:41.8464957+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.84701+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8473323+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8483922+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8497317+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8505057+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8511718+09:00 [INFO] Skip function Some("to_device_position_impl")
2024-12-08T13:24:41.8512944+09:00 [INFO] Skip function Some("to_device_position")
2024-12-08T13:24:41.8514055+09:00 [INFO] Skip function Some("to_device_position_transformed")
2024-12-08T13:24:41.8515158+09:00 [INFO] Skip function Some("to_tile_position")
2024-12-08T13:24:41.8517858+09:00 [INFO] Skip function Some("blend_color")
2024-12-08T13:24:41.8534369+09:00 [INFO] Using GPU: GPUSpecs { is_software_emulated: false, device_name: "NVIDIA GeForce GTX 1650 Ti", driver_name: "NVIDIA", driver_info: "566.14" }
2024-12-08T13:24:41.8550674+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.8582751+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.8610647+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.86307+09:00 [ERROR] theme not found: Catppuccin Mocha
2024-12-08T13:24:41.8927956+09:00 [INFO] Opening main db
2024-12-08T13:24:41.9029607+09:00 [INFO] Using surface present mode MAILBOX
2024-12-08T13:24:41.9068976+09:00 [WARN] Unable to forbid exclusive full screen
2024-12-08T13:24:41.9576078+09:00 [INFO] Using surface present mode MAILBOX
2024-12-08T13:24:41.9642475+09:00 [WARN] Unable to forbid exclusive full screen
2024-12-08T13:24:41.9815432+09:00 [INFO] Creating texture 0x12cb430ee30 of size 1024x1024x1 and format R8Unorm, name 'atlas', handle 0
2024-12-08T13:24:41.9863077+09:00 [INFO] Creating buffer 0x12cf961eb80 of size 65536, name 'chunk-0', handle 1
2024-12-08T13:24:42.0011197+09:00 [INFO] Creating buffer 0x12cf961e6d0 of size 4096, name 'chunk-0', handle 2
2024-12-08T13:24:42.0242111+09:00 [INFO] Using Windows API for pseudoconsole
2024-12-08T13:24:42.09989+09:00 [INFO] Creating buffer 0x12cf961ffd0 of size 6608, name 'chunk-1', handle 3
2024-12-08T13:24:42.1000365+09:00 [INFO] Creating buffer 0x12c811ad550 of size 4096, name 'chunk-2', handle 4
2024-12-08T13:24:42.1014773+09:00 [INFO] Creating buffer 0x12c811ae040 of size 4096, name 'chunk-3', handle 5
2024-12-08T13:24:42.101673+09:00 [INFO] Creating buffer 0x12c811ae1d0 of size 6608, name 'chunk-4', handle 6
2024-12-08T13:24:42.1018151+09:00 [INFO] Creating buffer 0x12c811ad6e0 of size 4096, name 'chunk-5', handle 7
2024-12-08T13:24:42.3195686+09:00 [INFO] add_connection;
2024-12-08T13:24:42.3318333+09:00 [INFO] set status on client 260961: Connected { peer_id: PeerId { owner_id: 620, id: 1492034 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-12-08T13:24:42.6886615+09:00 [INFO] Creating texture 0x12c810c25e0 of size 1024x1024x1 and format Bgra8UnormSrgb, name 'atlas', handle 8
2024-12-08T13:24:43.6092483+09:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "C:/", args: ["C:\\Users\\Kille\\AppData\\Local\\Zed\\copilot\\copilot-v0.7.0\\dist/language-server.js", "--stdio"]
2024-12-08T13:24:45.0520758+09:00 [INFO] Language server with id 0 sent unhandled notification window/logMessage:
{
  "type": 3,
  "message": "[lsp] GitHub Copilot Language Server 1.236.0 initialized"
}
2024-12-08T13:24:45.0528648+09:00 [INFO] Language server with id 0 sent unhandled notification workspace/configuration:
{
  "items": [
    {
      "section": "github.copilot"
    },
    {
      "section": "github-enterprise"
    },
    {
      "section": "http"
    },
    {
      "section": "telemetry"
    }
  ]
}
2024-12-08T13:24:45.0530004+09:00 [INFO] Language server with id 0 sent unhandled notification window/logMessage:
{
  "type": 3,
  "message": "[certificates] Removed 20 expired certificates"
}
2024-12-08T13:24:45.4447239+09:00 [INFO] Language server with id 0 sent unhandled notification featureFlagsNotification:
{
  "rt": false,
  "sn": false,
  "chat": true,
  "ic": true,
  "ep": false,
  "pc": false,
  "x": true
}
2024-12-08T13:24:45.444863+09:00 [INFO] Language server with id 0 sent unhandled notification conversation/preconditionsNotification:
{
  "results": [
    {
      "type": "chat_enabled",
      "status": "ok"
    }
  ],
  "status": "ok"
}
2024-12-08T13:24:48.1906614+09:00 [WARN] GPU frame is out of date
2024-12-08T13:24:48.1940816+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:48.2939661+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:49.9744172+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:49.9896584+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.0034887+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.033899+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.2893421+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.3034686+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.3197467+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.338085+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.3525687+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.3697117+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.3860875+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.4027576+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.4204847+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.4359582+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.4533072+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.4697865+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.4871215+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.5024962+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.519884+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.5359555+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.5525636+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.5708569+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.5874363+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.6026+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.6190777+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.6359439+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.6525893+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.6697043+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.6868847+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.7032287+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.7191475+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.7361573+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.7526226+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.7691438+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.7858858+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.8043289+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.8183833+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.8352304+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.8517164+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.8683706+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.8848959+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.9031509+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.9183502+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.935196+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.9516883+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.9684296+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:50.9868013+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.0019487+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.0184407+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.0351129+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.051903+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.0683217+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.0848342+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.1015421+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.1183468+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.1364698+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.1533841+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.1684821+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.1850768+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.2017766+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.2181481+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.2348489+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.2516777+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.2683545+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.2859722+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.3015847+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.3182532+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.3381262+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.3521491+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.368212+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.384883+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.4016889+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.4182734+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.4353239+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.4515097+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.4682535+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.4861865+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.5016537+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.5183582+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.5355221+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.5521509+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.5685859+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.5866136+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.6020938+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.6182887+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.6348648+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.6522005+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.6682012+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.685951+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.7032399+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.7183928+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.7352369+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.7516651+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.768242+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.7849747+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.8017291+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.8183752+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.8351127+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.8515737+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.8684484+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.8850176+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.9017586+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.9182671+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.9349508+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.9517542+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.968226+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:51.9863774+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.0015511+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.0184705+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.0371449+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.0516864+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.0684213+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.0851897+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.1045536+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.1183007+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.1350726+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.151711+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.1685125+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.1871521+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.2016993+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.2183931+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.2350522+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.2534766+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.2683105+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.2854797+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.302024+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.3200705+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.3388166+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.3518634+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.3684034+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.3850725+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.4017911+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.4203997+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.4350435+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.4517662+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.4684873+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.4872711+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.50201+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.5186405+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.5351061+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.5517023+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.5685353+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.5851644+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.6018067+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.6183618+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.6350169+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.6516736+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.6684484+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.6851815+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.7018906+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.7185036+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.7350368+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.7515601+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.7682979+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.7852755+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.8018712+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.8183788+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.8350087+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.8521561+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.8682041+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.8851729+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.9017149+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.9184259+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.9370658+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.9534462+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.9697358+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:52.986398+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.0021884+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.0196865+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.0349294+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.0516109+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.0684612+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.0851953+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.1017134+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.1184717+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.1353916+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.1521344+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.1688584+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.1852873+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.2017292+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.2184586+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.235167+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.252961+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.2684562+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.2850614+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.3017206+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.3188171+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.3352701+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.3550459+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.3686552+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.3866409+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4020824+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4185151+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4351886+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4515943+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4942215+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4944291+09:00 [INFO] Creating buffer 0x12cf961eea0 of size 26880, name 'chunk-6', handle 9
2024-12-08T13:24:53.4946383+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.4947287+09:00 [INFO] Creating buffer 0x12cf961fcb0 of size 6384, name 'chunk-7', handle 10
2024-12-08T13:24:53.4948535+09:00 [INFO] Creating buffer 0x12cf961f1c0 of size 26880, name 'chunk-8', handle 11
2024-12-08T13:24:53.5043725+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.5183092+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.5352424+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.5518986+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.5686017+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.5849963+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.6022132+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.6184387+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.6352418+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.6518966+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.6685412+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.6851391+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.7018341+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.7183156+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.7349526+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.7517621+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.7683615+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.7850304+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.8016983+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.8191158+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.834966+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.8516304+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.8683434+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.8849618+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.9016729+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:53.9184263+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.0399582+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.0518744+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.068511+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.0850663+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.1017678+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.1184036+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.1351353+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.1522671+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.1683541+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.1849538+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.2041756+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.219874+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.2349875+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.2516703+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.268346+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.2851237+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.301596+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.3183267+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.3349469+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.3548839+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.368323+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.3865252+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.4018968+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.4188566+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.4354251+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.4518356+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.468331+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.4851949+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.5018991+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.5193463+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.5349396+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.553366+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.5683387+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.5854857+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.6017868+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.6183829+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.6350512+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.6518934+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.668668+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.6852382+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.7018457+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.718466+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.7381988+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.7517633+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.7683778+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.7850962+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.8029575+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.8200893+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.8351009+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.851703+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.8684605+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.8856708+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.9017973+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.918803+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.9354347+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.9522496+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.9691182+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:54.9852941+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.0022756+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.0185383+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.0351255+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.0530165+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.0686357+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.0852377+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.10197+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.1186205+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.1358898+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.1518751+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.1684466+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.1849981+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.2020267+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.2187531+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.2351786+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.252259+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.268436+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.2851428+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.3018379+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.3184227+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.3352357+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.3547973+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.3688043+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.3851149+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.4018324+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.4185065+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.4351129+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.4519641+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.4683639+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.4850252+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.5017741+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.5189892+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.5359793+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.5534265+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.5699104+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.5851824+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.6023007+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.618359+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.6351494+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.6517458+09:00 [WARN] Acquire failed because the surface is out of date
2024-12-08T13:24:55.6574987+09:00 [INFO] Destroying texture 0x12cb430ee30, handle 0
2024-12-08T13:24:55.6575951+09:00 [INFO] Destroying texture 0x12c810c25e0, handle 8
2024-12-08T13:24:55.658219+09:00 [INFO] Destroying buffer 0x12cf961eb80, handle 1
2024-12-08T13:24:55.6585777+09:00 [INFO] Destroying buffer 0x12cf961e6d0, handle 2
2024-12-08T13:24:55.6586663+09:00 [INFO] Destroying buffer 0x12c811ae1d0, handle 6
2024-12-08T13:24:55.658736+09:00 [INFO] Destroying buffer 0x12cf961f1c0, handle 11
2024-12-08T13:24:55.6588126+09:00 [INFO] Destroying buffer 0x12cf961fcb0, handle 10
2024-12-08T13:24:55.6588814+09:00 [INFO] Destroying buffer 0x12cf961ffd0, handle 3
2024-12-08T13:24:55.6589498+09:00 [INFO] Destroying buffer 0x12c811ad550, handle 4
2024-12-08T13:24:55.6590179+09:00 [INFO] Destroying buffer 0x12c811ae040, handle 5
2024-12-08T13:24:55.6590862+09:00 [INFO] Destroying buffer 0x12c811ad6e0, handle 7
2024-12-08T13:24:55.6591598+09:00 [INFO] Destroying buffer 0x12cf961eea0, handle 9
```
</details>

> I have the same issue. Here's my log of opening zed, minimizing and bringing it back to front (frozen at this point), then closing it.
> 
> Zed.log
> ```
> 2024-12-08T13:24:41.1965219+09:00 [INFO] ========== starting zed ==========
> 2024-12-08T13:24:41.4100335+09:00 [INFO] Use Segoe UI as UI font.
> 2024-12-08T13:24:41.4115949+09:00 [INFO] Opening main db
> 2024-12-08T13:24:41.4203735+09:00 [INFO] Opening main db
> 2024-12-08T13:24:41.4304223+09:00 [INFO] Using git binary path: None
> 2024-12-08T13:24:41.4315947+09:00 [ERROR] SHELL environment variable is not assigned so we can't source login environment variables
> 
> Caused by:
>     environment variable not found
> 2024-12-08T13:24:41.4843874+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.4872606+09:00 [INFO] No prompt template overrides directory found at C:\Users\Kille\AppData\Local\Zed\prompt_overrides. Using built-in prompts.
> 2024-12-08T13:24:41.4879256+09:00 [INFO] extensions updated. loading 5, reloading 0, unloading 0
> 2024-12-08T13:24:41.4998366+09:00 [INFO] set status on client 0: Authenticating
> 2024-12-08T13:24:41.5002471+09:00 [INFO] set status on client 260961: Connecting
> 2024-12-08T13:24:41.5008604+09:00 [ERROR] Error { kind: Generic("Input watch path is neither a file nor a directory."), paths: [] }
> 2024-12-08T13:24:41.5019159+09:00 [ERROR] Error { kind: Generic("Input watch path is neither a file nor a directory."), paths: [] }
> 2024-12-08T13:24:41.5022281+09:00 [INFO] Opening main db
> 2024-12-08T13:24:41.5163412+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.5197882+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.5223621+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.5247584+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.5286404+09:00 [ERROR] Access is denied. (os error 5)
> 2024-12-08T13:24:41.6045264+09:00 [INFO] Enabling Vulkan Portability
> 2024-12-08T13:24:41.6046146+09:00 [INFO] Enabling color space support
> 2024-12-08T13:24:41.7085988+09:00 [INFO] Adapter: "NVIDIA GeForce GTX 1650 Ti"
> 2024-12-08T13:24:41.7087446+09:00 [INFO] No ray tracing extensions are supported
> 2024-12-08T13:24:41.7386884+09:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
> 2024-12-08T13:24:41.7890135+09:00 [WARN] Requested size 1x1 is outside of surface capabilities
> 2024-12-08T13:24:41.7891078+09:00 [ERROR] No composite alpha flag for transparency: OPAQUE
> 2024-12-08T13:24:41.7902281+09:00 [INFO] Using surface present mode MAILBOX
> 2024-12-08T13:24:41.7953429+09:00 [WARN] Unable to forbid exclusive full screen
> 2024-12-08T13:24:41.8110251+09:00 [INFO] Creating a descriptor pool for at most 16 sets
> 2024-12-08T13:24:41.8111251+09:00 [INFO] Creating a descriptor pool for at most 16 sets
> 2024-12-08T13:24:41.8112072+09:00 [INFO] Initializing Blade pipelines for surface SurfaceInfo { format: Bgra8UnormSrgb, alpha: Ignored }
> 2024-12-08T13:24:41.813871+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.814283+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8160233+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8164759+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8172176+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8175836+09:00 [INFO] Skip function Some("to_device_position_impl")
> 2024-12-08T13:24:41.8176667+09:00 [INFO] Skip function Some("to_device_position")
> 2024-12-08T13:24:41.8177437+09:00 [INFO] Skip function Some("to_device_position_transformed")
> 2024-12-08T13:24:41.8178206+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8179885+09:00 [INFO] Skip function Some("blend_color")
> 2024-12-08T13:24:41.8191999+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8198843+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8202354+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8210773+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8219437+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8225189+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8228425+09:00 [INFO] Skip function Some("to_device_position_impl")
> 2024-12-08T13:24:41.8229871+09:00 [INFO] Skip function Some("to_device_position")
> 2024-12-08T13:24:41.8230811+09:00 [INFO] Skip function Some("to_device_position_transformed")
> 2024-12-08T13:24:41.8231613+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8233474+09:00 [INFO] Skip function Some("blend_color")
> 2024-12-08T13:24:41.8275018+09:00 [ERROR] No composite alpha flag for transparency: OPAQUE
> 2024-12-08T13:24:41.8275968+09:00 [INFO] Using surface present mode MAILBOX
> 2024-12-08T13:24:41.830058+09:00 [WARN] Unable to forbid exclusive full screen
> 2024-12-08T13:24:41.8359167+09:00 [INFO] Using surface present mode MAILBOX
> 2024-12-08T13:24:41.8390576+09:00 [WARN] Unable to forbid exclusive full screen
> 2024-12-08T13:24:41.8408638+09:00 [INFO] Initializing Blade pipelines for surface SurfaceInfo { format: Bgra8UnormSrgb, alpha: Ignored }
> 2024-12-08T13:24:41.8432374+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8436056+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8442456+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8445842+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8451343+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8454468+09:00 [INFO] Skip function Some("to_device_position_impl")
> 2024-12-08T13:24:41.8455267+09:00 [INFO] Skip function Some("to_device_position")
> 2024-12-08T13:24:41.8456032+09:00 [INFO] Skip function Some("to_device_position_transformed")
> 2024-12-08T13:24:41.8456797+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8458306+09:00 [INFO] Skip function Some("blend_color")
> 2024-12-08T13:24:41.8464957+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.84701+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8473323+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8483922+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8497317+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8505057+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8511718+09:00 [INFO] Skip function Some("to_device_position_impl")
> 2024-12-08T13:24:41.8512944+09:00 [INFO] Skip function Some("to_device_position")
> 2024-12-08T13:24:41.8514055+09:00 [INFO] Skip function Some("to_device_position_transformed")
> 2024-12-08T13:24:41.8515158+09:00 [INFO] Skip function Some("to_tile_position")
> 2024-12-08T13:24:41.8517858+09:00 [INFO] Skip function Some("blend_color")
> 2024-12-08T13:24:41.8534369+09:00 [INFO] Using GPU: GPUSpecs { is_software_emulated: false, device_name: "NVIDIA GeForce GTX 1650 Ti", driver_name: "NVIDIA", driver_info: "566.14" }
> 2024-12-08T13:24:41.8550674+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.8582751+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.8610647+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.86307+09:00 [ERROR] theme not found: Catppuccin Mocha
> 2024-12-08T13:24:41.8927956+09:00 [INFO] Opening main db
> 2024-12-08T13:24:41.9029607+09:00 [INFO] Using surface present mode MAILBOX
> 2024-12-08T13:24:41.9068976+09:00 [WARN] Unable to forbid exclusive full screen
> 2024-12-08T13:24:41.9576078+09:00 [INFO] Using surface present mode MAILBOX
> 2024-12-08T13:24:41.9642475+09:00 [WARN] Unable to forbid exclusive full screen
> 2024-12-08T13:24:41.9815432+09:00 [INFO] Creating texture 0x12cb430ee30 of size 1024x1024x1 and format R8Unorm, name 'atlas', handle 0
> 2024-12-08T13:24:41.9863077+09:00 [INFO] Creating buffer 0x12cf961eb80 of size 65536, name 'chunk-0', handle 1
> 2024-12-08T13:24:42.0011197+09:00 [INFO] Creating buffer 0x12cf961e6d0 of size 4096, name 'chunk-0', handle 2
> 2024-12-08T13:24:42.0242111+09:00 [INFO] Using Windows API for pseudoconsole
> 2024-12-08T13:24:42.09989+09:00 [INFO] Creating buffer 0x12cf961ffd0 of size 6608, name 'chunk-1', handle 3
> 2024-12-08T13:24:42.1000365+09:00 [INFO] Creating buffer 0x12c811ad550 of size 4096, name 'chunk-2', handle 4
> 2024-12-08T13:24:42.1014773+09:00 [INFO] Creating buffer 0x12c811ae040 of size 4096, name 'chunk-3', handle 5
> 2024-12-08T13:24:42.101673+09:00 [INFO] Creating buffer 0x12c811ae1d0 of size 6608, name 'chunk-4', handle 6
> 2024-12-08T13:24:42.1018151+09:00 [INFO] Creating buffer 0x12c811ad6e0 of size 4096, name 'chunk-5', handle 7
> 2024-12-08T13:24:42.3195686+09:00 [INFO] add_connection;
> 2024-12-08T13:24:42.3318333+09:00 [INFO] set status on client 260961: Connected { peer_id: PeerId { owner_id: 620, id: 1492034 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
> 2024-12-08T13:24:42.6886615+09:00 [INFO] Creating texture 0x12c810c25e0 of size 1024x1024x1 and format Bgra8UnormSrgb, name 'atlas', handle 8
> 2024-12-08T13:24:43.6092483+09:00 [INFO] starting language server process. binary path: "C:\\Program Files\\nodejs\\node.exe", working directory: "C:/", args: ["C:\\Users\\Kille\\AppData\\Local\\Zed\\copilot\\copilot-v0.7.0\\dist/language-server.js", "--stdio"]
> 2024-12-08T13:24:45.0520758+09:00 [INFO] Language server with id 0 sent unhandled notification window/logMessage:
> {
>   "type": 3,
>   "message": "[lsp] GitHub Copilot Language Server 1.236.0 initialized"
> }
> 2024-12-08T13:24:45.0528648+09:00 [INFO] Language server with id 0 sent unhandled notification workspace/configuration:
> {
>   "items": [
>     {
>       "section": "github.copilot"
>     },
>     {
>       "section": "github-enterprise"
>     },
>     {
>       "section": "http"
>     },
>     {
>       "section": "telemetry"
>     }
>   ]
> }
> 2024-12-08T13:24:45.0530004+09:00 [INFO] Language server with id 0 sent unhandled notification window/logMessage:
> {
>   "type": 3,
>   "message": "[certificates] Removed 20 expired certificates"
> }
> 2024-12-08T13:24:45.4447239+09:00 [INFO] Language server with id 0 sent unhandled notification featureFlagsNotification:
> {
>   "rt": false,
>   "sn": false,
>   "chat": true,
>   "ic": true,
>   "ep": false,
>   "pc": false,
>   "x": true
> }
> 2024-12-08T13:24:45.444863+09:00 [INFO] Language server with id 0 sent unhandled notification conversation/preconditionsNotification:
> {
>   "results": [
>     {
>       "type": "chat_enabled",
>       "status": "ok"
>     }
>   ],
>   "status": "ok"
> }
> 2024-12-08T13:24:48.1906614+09:00 [WARN] GPU frame is out of date
> 2024-12-08T13:24:48.1940816+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:48.2939661+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:49.9744172+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:49.9896584+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.0034887+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.033899+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.2893421+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.3034686+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.3197467+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.338085+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.3525687+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.3697117+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.3860875+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.4027576+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.4204847+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.4359582+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.4533072+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.4697865+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.4871215+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.5024962+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.519884+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.5359555+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.5525636+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.5708569+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.5874363+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.6026+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.6190777+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.6359439+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.6525893+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.6697043+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.6868847+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.7032287+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.7191475+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.7361573+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.7526226+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.7691438+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.7858858+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.8043289+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.8183833+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.8352304+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.8517164+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.8683706+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.8848959+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.9031509+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.9183502+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.935196+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.9516883+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.9684296+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:50.9868013+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.0019487+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.0184407+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.0351129+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.051903+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.0683217+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.0848342+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.1015421+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.1183468+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.1364698+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.1533841+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.1684821+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.1850768+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.2017766+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.2181481+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.2348489+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.2516777+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.2683545+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.2859722+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.3015847+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.3182532+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.3381262+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.3521491+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.368212+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.384883+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.4016889+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.4182734+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.4353239+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.4515097+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.4682535+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.4861865+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.5016537+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.5183582+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.5355221+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.5521509+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.5685859+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.5866136+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.6020938+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.6182887+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.6348648+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.6522005+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.6682012+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.685951+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.7032399+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.7183928+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.7352369+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.7516651+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.768242+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.7849747+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.8017291+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.8183752+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.8351127+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.8515737+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.8684484+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.8850176+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.9017586+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.9182671+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.9349508+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.9517542+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.968226+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:51.9863774+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.0015511+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.0184705+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.0371449+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.0516864+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.0684213+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.0851897+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.1045536+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.1183007+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.1350726+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.151711+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.1685125+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.1871521+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.2016993+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.2183931+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.2350522+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.2534766+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.2683105+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.2854797+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.302024+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.3200705+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.3388166+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.3518634+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.3684034+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.3850725+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.4017911+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.4203997+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.4350435+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.4517662+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.4684873+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.4872711+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.50201+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.5186405+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.5351061+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.5517023+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.5685353+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.5851644+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.6018067+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.6183618+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.6350169+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.6516736+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.6684484+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.6851815+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.7018906+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.7185036+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.7350368+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.7515601+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.7682979+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.7852755+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.8018712+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.8183788+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.8350087+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.8521561+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.8682041+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.8851729+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.9017149+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.9184259+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.9370658+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.9534462+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.9697358+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:52.986398+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.0021884+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.0196865+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.0349294+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.0516109+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.0684612+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.0851953+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.1017134+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.1184717+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.1353916+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.1521344+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.1688584+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.1852873+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.2017292+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.2184586+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.235167+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.252961+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.2684562+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.2850614+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.3017206+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.3188171+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.3352701+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.3550459+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.3686552+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.3866409+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4020824+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4185151+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4351886+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4515943+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4942215+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4944291+09:00 [INFO] Creating buffer 0x12cf961eea0 of size 26880, name 'chunk-6', handle 9
> 2024-12-08T13:24:53.4946383+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.4947287+09:00 [INFO] Creating buffer 0x12cf961fcb0 of size 6384, name 'chunk-7', handle 10
> 2024-12-08T13:24:53.4948535+09:00 [INFO] Creating buffer 0x12cf961f1c0 of size 26880, name 'chunk-8', handle 11
> 2024-12-08T13:24:53.5043725+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.5183092+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.5352424+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.5518986+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.5686017+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.5849963+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.6022132+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.6184387+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.6352418+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.6518966+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.6685412+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.6851391+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.7018341+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.7183156+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.7349526+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.7517621+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.7683615+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.7850304+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.8016983+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.8191158+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.834966+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.8516304+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.8683434+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.8849618+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.9016729+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:53.9184263+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.0399582+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.0518744+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.068511+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.0850663+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.1017678+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.1184036+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.1351353+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.1522671+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.1683541+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.1849538+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.2041756+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.219874+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.2349875+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.2516703+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.268346+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.2851237+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.301596+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.3183267+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.3349469+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.3548839+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.368323+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.3865252+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.4018968+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.4188566+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.4354251+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.4518356+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.468331+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.4851949+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.5018991+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.5193463+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.5349396+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.553366+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.5683387+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.5854857+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.6017868+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.6183829+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.6350512+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.6518934+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.668668+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.6852382+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.7018457+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.718466+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.7381988+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.7517633+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.7683778+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.7850962+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.8029575+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.8200893+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.8351009+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.851703+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.8684605+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.8856708+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.9017973+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.918803+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.9354347+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.9522496+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.9691182+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:54.9852941+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.0022756+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.0185383+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.0351255+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.0530165+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.0686357+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.0852377+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.10197+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.1186205+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.1358898+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.1518751+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.1684466+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.1849981+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.2020267+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.2187531+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.2351786+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.252259+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.268436+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.2851428+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.3018379+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.3184227+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.3352357+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.3547973+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.3688043+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.3851149+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.4018324+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.4185065+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.4351129+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.4519641+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.4683639+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.4850252+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.5017741+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.5189892+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.5359793+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.5534265+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.5699104+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.5851824+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.6023007+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.618359+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.6351494+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.6517458+09:00 [WARN] Acquire failed because the surface is out of date
> 2024-12-08T13:24:55.6574987+09:00 [INFO] Destroying texture 0x12cb430ee30, handle 0
> 2024-12-08T13:24:55.6575951+09:00 [INFO] Destroying texture 0x12c810c25e0, handle 8
> 2024-12-08T13:24:55.658219+09:00 [INFO] Destroying buffer 0x12cf961eb80, handle 1
> 2024-12-08T13:24:55.6585777+09:00 [INFO] Destroying buffer 0x12cf961e6d0, handle 2
> 2024-12-08T13:24:55.6586663+09:00 [INFO] Destroying buffer 0x12c811ae1d0, handle 6
> 2024-12-08T13:24:55.658736+09:00 [INFO] Destroying buffer 0x12cf961f1c0, handle 11
> 2024-12-08T13:24:55.6588126+09:00 [INFO] Destroying buffer 0x12cf961fcb0, handle 10
> 2024-12-08T13:24:55.6588814+09:00 [INFO] Destroying buffer 0x12cf961ffd0, handle 3
> 2024-12-08T13:24:55.6589498+09:00 [INFO] Destroying buffer 0x12c811ad550, handle 4
> 2024-12-08T13:24:55.6590179+09:00 [INFO] Destroying buffer 0x12c811ae040, handle 5
> 2024-12-08T13:24:55.6590862+09:00 [INFO] Destroying buffer 0x12c811ad6e0, handle 7
> 2024-12-08T13:24:55.6591598+09:00 [INFO] Destroying buffer 0x12cf961eea0, handle 9
> ```

not sure why I am unable to replicate it
[Zed.log](https://github.com/user-attachments/files/18050794/Zed.log)

--- a/crates/gpui/src/platform/blade/blade_renderer.rs
+++ b/crates/gpui/src/platform/blade/blade_renderer.rs
@@ -431,13 +431,25 @@ impl BladeRenderer {
     }
 
     pub fn update_drawable_size(&mut self, size: Size<DevicePixels>) {
+        self.update_drawable_size_impl(size, false);
+    }
+
+    /// Like `update_drawable_size` but skips the check that the size has changed. This is useful in
+    /// cases like restoring a window from minimization where the size is the same but the
+    /// renderer's swap chain needs to be recreated.
+    #[cfg_attr(any(target_os = "macos", target_os = "linux"), allow(dead_code))]
+    pub fn update_drawable_size_even_if_unchanged(&mut self, size: Size<DevicePixels>) {
+        self.update_drawable_size_impl(size, true);
+    }
+
+    fn update_drawable_size_impl(&mut self, size: Size<DevicePixels>, always_resize: bool) {
         let gpu_size = gpu::Extent {
             width: size.width.0 as u32,
             height: size.height.0 as u32,
             depth: 1,
         };
 
-        if gpu_size != self.surface_config.size {
+        if always_resize || gpu_size != self.surface_config.size {
             self.wait_for_gpu();
             self.surface_config.size = gpu_size;
             self.gpu.resize(self.surface_config);

--- a/crates/gpui/src/platform/windows/events.rs
+++ b/crates/gpui/src/platform/windows/events.rs
@@ -141,16 +141,27 @@ fn handle_size_msg(
     lparam: LPARAM,
     state_ptr: Rc<WindowsWindowStatePtr>,
 ) -> Option<isize> {
+    let mut lock = state_ptr.state.borrow_mut();
+
+    // Don't resize the renderer when the window is minimized, but record that it was minimized so
+    // that on restore the swap chain can be recreated via `update_drawable_size_even_if_unchanged`.
     if wparam.0 == SIZE_MINIMIZED as usize {
+        lock.is_minimized = Some(true);
         return Some(0);
     }
+    let may_have_been_minimized = lock.is_minimized.unwrap_or(true);
+    lock.is_minimized = Some(false);
 
     let width = lparam.loword().max(1) as i32;
     let height = lparam.hiword().max(1) as i32;
-    let mut lock = state_ptr.state.borrow_mut();
     let new_size = size(DevicePixels(width), DevicePixels(height));
     let scale_factor = lock.scale_factor;
-    lock.renderer.update_drawable_size(new_size);
+    if may_have_been_minimized {
+        lock.renderer
+            .update_drawable_size_even_if_unchanged(new_size);
+    } else {
+        lock.renderer.update_drawable_size(new_size);
+    }
     let new_size = new_size.to_pixels(scale_factor);
     lock.logical_size = new_size;
     if let Some(mut callback) = lock.callbacks.resize.take() {

--- a/crates/gpui/src/platform/windows/window.rs
+++ b/crates/gpui/src/platform/windows/window.rs
@@ -38,6 +38,7 @@ pub struct WindowsWindowState {
     pub fullscreen_restore_bounds: Bounds<Pixels>,
     pub border_offset: WindowBorderOffset,
     pub scale_factor: f32,
+    pub is_minimized: Option<bool>,
 
     pub callbacks: Callbacks,
     pub input_handler: Option<PlatformInputHandler>,
@@ -92,6 +93,7 @@ impl WindowsWindowState {
             size: logical_size,
         };
         let border_offset = WindowBorderOffset::default();
+        let is_minimized = None;
         let renderer = windows_renderer::windows_renderer(hwnd, transparent)?;
         let callbacks = Callbacks::default();
         let input_handler = None;
@@ -109,6 +111,7 @@ impl WindowsWindowState {
             fullscreen_restore_bounds,
             border_offset,
             scale_factor,
+            is_minimized,
             callbacks,
             input_handler,
             system_key_handled,

